app/config/collections.php

```diff
@@ -3580,6 +3580,139 @@
             ],
         ],
     ],
+
+    'imports' => [
+        '$collection' => ID::custom(Database::METADATA),
+        '$id' => ID::custom('imports'),
+        'name' => 'Imports',
+        'attributes' => [
+            [
+                '$id' => ID::custom('status'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => true,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('stage'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => true,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('source'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 8192,
+                'signed' => true,
+                'required' => true,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('resources'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => true,
+                'default' => [],
+                'array' => true,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('statusCounters'),
+                'type' => Database::VAR_STRING,
```

Comment:
Apply proper changes according to changes required in the response model. Both structures should be pretty similar.